x/staging/posts/keeper/grpc_query_test.go

```diff
@@ -457,8 +457,8 @@ func (suite *KeeperTestSuite) Test_UserAnswers() {
 		},
 	}
 
-	suite.SetupTest()
 	for _, uc := range usecases {
+		suite.SetupTest()
```

Comment:
The cache context is used in the test. I think `SetupTest()` could be outside to make sure initialization.

Reply:
Oh this is unwanted, reverted it now